### PR TITLE
Validate multi config

### PR DESCRIPTION
**Pull request by tehpsalmist**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -40,6 +51,35 @@ describe('module', function () {
...
+				'x-zengine-webhook-key': 456
+			},
+			body: {
+				webhook: {
+					id: 987
+				}
+			}
+		};
+
+		expect($webhook.getValidatedMultiConfigId(multiConfigSettings, reqWithBadId)).to.equal(undefined);
```
Comment (alexweber) [inline test/test.js:78]: @tehpsalmist im almost certain there's a "better" version which is`expect(foo).to.be.undefined`

Comment (tehpsalmist) [reply test/test.js:78]: Well what do ya know.

**Comment by tehpsalmist** [comment]
Hey @alexweber or @AParks Can this be merged or something? I don't think these checks are working.

**Comment by alexweber** [comment]
@tehpsalmist lets discuss it real quick then ill merge

**Pull request merged**